### PR TITLE
Replace R8 binary with maven dependency

### PR DESCRIPTION
**Pull request by Goooler**
```
OLD: vgo.before
NEW: vgo.after

 JAR   │ old │ new │ diff
───────┼─────┼─────┼──────
 class │ 0 B │ 0 B │  0 B
 other │ 0 B │ 0 B │  0 B
───────┼─────┼─────┼──────
 total │ 0 B │ 0 B │  0 B

 CLASSES │ old │ new │ diff
─────────┼─────┼─────┼───────────
 classes │   0 │   0 │ 0 (+0 -0)
 methods │   0 │   0 │ 0 (+0 -0)
  fields │   0 │   0 │ 0 (+0 -0)
```

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -9,6 +9,13 @@ dependencyResolutionManagement {
     repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
     repositories {
         mavenCentral()
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
```
Comment (jzbrooks) [inline settings.gradle.kts:17]: Is there a problem/conflict including the entire google maven repo (removing the constraints)?

Comment (Goooler) [reply settings.gradle.kts:17]: Nope. Add these regex will speed up downloading.

Comment (jzbrooks) [reply settings.gradle.kts:17]: By how much? I lean toward the simpler change, but I could be convinced if it was a big perf swing.

Comment (Goooler) [reply settings.gradle.kts:17]: No much effects for small projects. I can revert this if you want.

**Comment by jzbrooks** [comment]
Thanks!

**Pull request merged**